frontend/asset_mapper.rst

```diff
@@ -289,17 +289,25 @@ Preloading and Initializing "app.js"
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In addition to the importmap, the ``{{ importmap() }}`` Twig function also renders
-an `ES module shim`_ and a few other things, like a set of "preloads":
+an `ES module shim`_. You can disable this in your ``config/packages/asset_mapper.yaml``
+configuration file:
 
-.. code-block:: html
+.. code-block:: yaml
 
-    <link rel="modulepreload" href="/assets/app-4e986c1a2318dd050b1d47db8d856278.js">
-    <link rel="modulepreload" href="/assets/duck-1b7a64b3b3d31219c262cf72521a5267.js">
+    asset_mapper:
```

Comment:
This misses the `framework:` top-level key for the FrameworkBundle config. `asset_mapper` is configured as part of FrameworkBundle, not as a dedicated AssetMapperBundle